devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -96,16 +104,16 @@ def refresh() {
     zigbee.onOffRefresh() + zigbee.levelRefresh()
 }
 
+def configureHealthCheck() {
+    log.debug "configureHealthCheck"
+    unschedule("healthPoll")
+    runEvery5Minutes("healthPoll")
+    // Device-Watch allows 2 check-in misses from device
+    sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+}
+
 def healthPoll() {
     log.debug "healthPoll()"
     def cmds = zigbee.onOffRefresh() + zigbee.levelRefresh()
     cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
 }
-
-def configure() {
-    unschedule()
-    runEvery5Minutes("healthPoll")
-    // Device-Watch allows 2 check-in misses from device
-    sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
-    zigbee.onOffRefresh() + zigbee.levelRefresh()
```

Comment:
I think we probably do want to keep `configure()` just so that it can call `zigbee.onOffRefresh() + zigbee.levelRefresh()`. Otherwise the those events won't get created until on/off/setLevel is called